### PR TITLE
add release-notes for 2.0.0

### PR DESCRIPTION
**Pull request by raunakkumar**
Signed-off-by: Raunak <raunak.kumar@hpe.com>

### CODE REVIEW

File: release-notes/v2.0.0.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.0.0.
+
+## Known Issues
+
+The following table lists the known issues for HPE CSI Driver for Kubernetes v2.0.0. Please note that, `Known Issues` from previous releases are still applicable with suggested workarounds, if they are not part of `Resolved Issues` above.
+
+|ID|Component |Title|Description|Workaround|
+|--|---------|-----|-----------|----------|
+|CON-1821|csi.k8s|Not able to create initiatorgroup for long hostname|initiator group name is limited to 60 characters.|Limit the hostname to lesser than 60 characters|
```
Comment (rgcostea) [inline release-notes/v2.0.0.md:19]:
```suggestion
|CON-1821|csi.k8s|Not able to create initiator group for long hostname|initiator group name is limited to 60 characters.|Limit the hostname to lesser than 60 characters|
```

File: release-notes/v2.0.0.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+|CON-2013|csi.k8s|"btrfs" filesystem limitation - it does not use inodes as other filesystems|inodes metrics as part of NodeGetVolumeStats are not present for btrfs filesystem. https://unix.stackexchange.com/questions/525637/debian-btrfs-filesystem-shows-a-total-of-zero-inodes-total-zero-used-zero-free| None|
+|CON-2109|csi.k8s|Scale setup with mixed workloads may bottlenecked the CSP|In a large scale environment when the PVCs and associated pods are created at once, it creates a large number of requests to the CSP.|Create the pvcs prior to creating the workload to lessen the load on the CSP.|
```
Comment (rgcostea) [inline release-notes/v2.0.0.md:21]:
```suggestion
|CON-2109|csi.k8s|Scale setup with mixed workloads may bottleneck the CSP|In a large scale environment when the PVCs and associated pods are created at once, it creates a large number of requests to the CSP.|Create the PVCs prior to creating the workload to lessen the load on the CSP.|
```

File: release-notes/v2.0.0.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+|CON-2116|csi.k8s| Volume with xfs filesystem created on Ubuntu does not mount on CentOS| xfs Filesystem created on Ubuntu 20.04 fails to mount on another CentOS worker node with error wrong fs type, bad option, bad superblock. |None|
+|CON-2121|csi.k8s|Ephemeral volumes are not deleted after node shutdown|Ephemeral volumes are created as part of NodePublish. On node shutdown if the pods are deleted after MaxWaitForUnmount, the scrubber does not clean up orphan volumes have no pods leaving in orphan volumes|None|
```
Comment (rgcostea) [inline release-notes/v2.0.0.md:23]: > the scrubber does not clean up orphan volumes have no pods leaving in orphan volumes

Consider rewording

File: release-notes/v2.0.0.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.0.0.
+
+## Known Issues
+
+The following table lists the known issues for HPE CSI Driver for Kubernetes v2.0.0. Please note that, `Known Issues` from previous releases are still applicable with suggested workarounds, if they are not part of `Resolved Issues` above.
+
+|ID|Component |Title|Description|Workaround|
+|--|---------|-----|-----------|----------|
+|CON-1821|csi.k8s|Not able to create initiatorgroup for long hostname|initiator group name is limited to 60 characters.|Limit the hostname to lesser than 60 characters|
+|CON-2013|csi.k8s|"btrfs" filesystem limitation - it does not use inodes as other filesystems|inodes metrics as part of NodeGetVolumeStats are not present for btrfs filesystem. https://unix.stackexchange.com/questions/525637/debian-btrfs-filesystem-shows-a-total-of-zero-inodes-total-zero-used-zero-free| None|
```
Comment (rgcostea) [inline release-notes/v2.0.0.md:20]:
```suggestion
|CON-2013|csi.k8s|"btrfs" filesystem limitation - it does not use inodes as other filesystems|inode metrics as part of NodeGetVolumeStats are not present for btrfs filesystem. https://unix.stackexchange.com/questions/525637/debian-btrfs-filesystem-shows-a-total-of-zero-inodes-total-zero-used-zero-free| None|
```

File: release-notes/v2.0.0.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.0.0.
+
+## Known Issues
+
+The following table lists the known issues for HPE CSI Driver for Kubernetes v2.0.0. Please note that, `Known Issues` from previous releases are still applicable with suggested workarounds, if they are not part of `Resolved Issues` above.
+
+|ID|Component |Title|Description|Workaround|
+|--|---------|-----|-----------|----------|
+|CON-1821|csi.k8s|Not able to create initiator group for long hostname|initiator group name is limited to 60 characters.|Limit the hostname to lesser than 60 characters|
```
Comment (suneeth51) [inline release-notes/v2.0.0.md:19]: nit: Initiator (upper case)

File: release-notes/v2.0.0.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+|CON-2013|csi.k8s|"btrfs" filesystem limitation - it does not use inodes as other filesystems|inode metrics as part of NodeGetVolumeStats are not present for btrfs filesystem. https://unix.stackexchange.com/questions/525637/debian-btrfs-filesystem-shows-a-total-of-zero-inodes-total-zero-used-zero-free| None|
```
Comment (suneeth51) [inline release-notes/v2.0.0.md:20]: nit: Inode (upper case)

**Review by suneeth51 — APPROVED** [review]
lgtm.. with minor comments

**Pull request merged**